network/benchmarks/netperf/nptest/Dockerfile

```diff
@@ -31,6 +31,8 @@ RUN go build -o nptests
 FROM debian:bullseye
 ENV LD_LIBRARY_PATH=/usr/local/lib
 
+LABEL org.opencontainers.image.description "Network performance tests in k8s engine"
```

Comment:
what is this for?

Reply:
It adds a description to ghcr package page